### ISSUE TITLE
sync to external audio

### ISSUE THREAD

**Issue by ghost** [joaquintorres/autosubsync-mpv#28]
I set audio-file-auto=fuzzy and put audio and video same location, but autosubsync can't sync with both alass and ffsubsync it feels like it still use internal audio, I use external audio cuz its filtered.

**Comment by tatsumoto-ren** [comment]
I don't think we have implemented this feature since it's pretty rare that people use external audio tracks. External subtitle tracks are more widely used.
If you're interested in working on the implementation, let me know.

**Comment by tatsumoto-ren** [comment]
gonna keep it open in case we have time to work on the feature.
